test-cases/artifacts/amazon2.yaml

```diff
@@ -0,0 +1,20 @@
+ami_db_scylla_user: 'ec2-user'
```

Comment:
can you this configuration file ? it's not related those changes

Reply:
same as my comment above...

Reply:
same as the comment above ;)